installation/ilmt-track-bamlcns-kubernetes.adoc

```diff
@@ -0,0 +1,37 @@
+= Tracking BAMOE license use in Kubernetes
+include::../partials/attributes.adoc[]
+
+
+== Prerequisites
+Install the License Service by following the https://www.ibm.com/docs/en/cloud-paks/foundational-services/3.23?topic=software-installing-license-service-stand-alone[tutorial].
+
+== Tracking BAMOE License
+After installing BAMOE Canvas and its dependencies following the installation tutorial, run the following commands depending on where BAMOE Canvas was installed.
+
+* OpenShift
+----
+oc patch deployment <bamoe-canvas_deployment_name> -p \
+  '{"spec": {"template":{"metadata":{"annotations":{"productID":"46b14c7f2f894218a7879b3f6416024f", "productName": "IBM Decision Manager Open Edition", "productMetric": "VIRTUAL_PROCESSOR_CORE", "productChargedContainers": "All"}}}} }'
+----
+  
+* Kubernetes
+----
+kubectl patch deployment <bamoe-canvas_deployment_name> -p \
+  '{"spec": {"template":{"metadata":{"annotations":{"productID":"46b14c7f2f894218a7879b3f6416024f", "productName": "IBM Decision Manager Open Edition", "productMetric": "VIRTUAL_PROCESSOR_CORE", "productChargedContainers": "All"}}}} }'
+----
+  Where <bamoe-canvas_deployment_name> is the name of the Deployment resource created when you installed BAMOE Canvas.
```

Comment:
Probably use `<bamoe-canvas_deployment_name>` (in backticks) to set it off better.